_search-plugins/star-tree-index.md

```diff
@@ -185,6 +197,48 @@ POST /logs/_search
 
 Using a star-tree index, the result will be retrieved from a single aggregated document as it traverses the `status=500` node, as opposed to scanning through all of the matching documents. This results in lower query latency.
 
+### Date histogram with metric aggregations
+
+You can use [date histograms]({{site.url}}{{site.baseurl}}/aggregations/bucket/date-histogram/) on calendar intervals with metric sub-aggregations.
+
+To use date histogram aggregations and make then searchable in the star-tree index, use the following steps:
+
+- The calendar intervals in star-tree mapping configuration should have either the request calendar field or a lower granularity calendar field. For example, `month` can be resolved by the star-tree from `day` field as well if present in star-tree mapping.
```

Comment:
I don't understand this interval example. Are we saying `month` and `day` can be used in the same calendar interval and both be searchable from the star-tree?

```suggestion
- The calendar intervals in star-tree mapping configuration should have either the request calendar field or a lower granularity calendar field. For example, `month` can be resolved by the star-tree from `day` field as well if present in star-tree mapping.
```

Reply:
Basically we are saying that for example, star-tree mapping has `minute` & `day` as indexed timestamp fields. So calculating `minute` and `day` aggregation from star-tree will be a trivial supported case. What the example emphasizes that even `month` or `year` aggregation can be supported by sta-tree search by making use of lower granularity intervals like `day` in this case. Similarly if `hour` aggregation is asked, it can make use of `minute` field in star-tree to compute that. But let's say second granularity is asked, it will not use star-tree as `second` or a lower granularity is not present and the search code flow will default to default non-star-tree approach.